admin/translations/fr.json

```
@@ -137,6 +137,12 @@
           "tabName": "Déclencheurs"
         }
       }
+    },
+    "user-details": {
+      "navbar": {
+        "organizations-list": "Pix Orga",
+        "certification-centers-list": "Pix Certif"
```

les clés doivent être triés par ordre alphabétique (pareil pour le fichier `en.json`) 😄 
```suggestion
        "certification-centers-list": "Pix Certif",
        "organizations-list": "Pix Orga"
```